### PR TITLE
Release 5.0.2

### PR DESCRIPTION
**Pull request by thiemowmde**

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,8 +1,12 @@
 # Wikibase DataModel release notes
 
+## Version 5.0.2 (2016-02-23)
+
+* Fixed regression in `ReferenceList::addReference` and the constructor possibly adding to many objects
+
 ## Version 5.0.1 (2016-02-18)
 
-* Fixed a bug and restored behaviour of `ReferenceList::removeReferenceHash`
+* Fixed regression in `ReferenceList::removeReferenceHash` possibly removing to many objects
```
Comment (Benestar) [inline RELEASE-NOTES.md]: too many

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,8 +1,12 @@
 # Wikibase DataModel release notes
 
+## Version 5.0.2 (2016-02-23)
+
+* Fixed regression in `ReferenceList::addReference` and the constructor possibly adding to many objects
```
Comment (Benestar) [inline RELEASE-NOTES.md]: too many

**Comment by thiemowmde** [comment]
@adrianheine, do you want to add #626 to this release? We can also move this to a possible 5.0.3 or 5.1 if you are not sure.

**Comment by JeroenDeDauw** [comment]
#626 does not seem like a bugfix, in which case you cannot include it here

**Comment by adrianheine** [comment]
Bene just wrote 5.1.

**Comment by thiemowmde** [comment]
Well, then let's release 5.0.2 without #626. No problem.

**Pull request merged**